### DISCUSSION TITLE
Getting Locked Out despite having Admin access

### DISCUSSION THREAD

**Discussion by ManuWeb3** [GitbookIO/community#361] [0. General bugs and questions] [answered]
Hi Gitbooik team,

I have admin access to my organization's gitbook handle.
Actually, I just received one and I'm new to Gitbook.

![image](https://github.com/GitbookIO/community/assets/108497781/1161f4cb-18cd-4da8-9b15-5e39345444c7)

But, I'm getting "Locked out" everytime I access any of the tabs on the sidebar of our org.

![image](https://github.com/GitbookIO/community/assets/108497781/ffaa8016-d8e3-43ff-9d06-1e79c14ed83e)

I went through the discussion at the thread below as it says some issue with the computation of permissions:
https://github.com/orgs/GitbookIO/discussions/328

Also, cleared the entire Cache as suggested + changed the browser also and re-tried, but to no avail.

Kindly help.

We're a startup and reaching our Production Launch pretty soon. 

Can't afford any delays with the Docs.

Thank you
Manu

**Comment by SamyPesse** [comment, marked as answer]
Hey @ManuWeb3, for support, you should contact our support team at support@gitbook.com (or from the app).

But I quickly looked into it, and it looks like you are not an admin in the org (you have only "Editor" permissions), but Julian is. Can you reach out to them to ask them to upgrade?

**Reply by ManuWeb3** [reply]
Hey @SamyPesse , thanks for taking the time to reply and looking into it.

I did send an email just before posting the question here. I wasn't sure of getting a reply due to weekend, hence thought of posting it here as well.

Sure, I'll reach out to Julian for an upgrade.

Wondering how come I only have Editor permission when I show up as an Admin.

Anyway, probably, I need to look more into Gitbook's docs :)

Thanks
Manu

**Reply by SamyPesse** [reply]
Where does the screenshot where you appear as an admin comes from?

You might be an admin on the space but not on the organization.

**Reply by ManuWeb3** [reply]
Hey @SamyPesse , thanks for the follow up.

![SS1](https://github.com/GitbookIO/community/assets/108497781/ef0f5da0-20ee-416d-9ea0-24a480c880b0)

![SS2](https://github.com/GitbookIO/community/assets/108497781/dc541965-1b07-42bc-a98b-d1bd0d53784d)


Does "Administrator" here show that I'm admin of this space only?

=====================

**(1).** By the way, you said I'm an Editor. So, why can't I login and being asked for an Upgrade ?

Being an Editor, I should at least be able to login and make edits but I'm blocked altogether.

**(2).** The docs talk about an Admin at the organnization level, not at the space level unlike what you said.

![image](https://github.com/GitbookIO/community/assets/108497781/0a3e09db-95f0-4d3e-a7f5-95a99bf9baf5)

What am I missing here?

**Reply by SamyPesse** [reply]
>Does "Administrator" here show that I'm admin of this space only?

Yes, it represents your permission at the space level. But it doesn't mean you are an administrator at the organization level. We'll look at clarifying this in the UI.

> By the way, you said I'm an Editor. So, why can't I login and being asked for an Upgrade ?

The organization is frozen because the 14-day free trial is over. An administrator of the organization needs to upgrade it to a paying plan or you need to switch to the free plan (in that case you'll need to remove members of the organization as you currently have 3 members).

**Reply by ManuWeb3** [reply]
Thanks @SamyPesse for consistently asnwering my questions.
